integration/src/main/java/scripts/testscripts/ControlledGcsBucketLifecycle.java

```diff
@@ -202,18 +202,23 @@ public void doUserJourney(TestUserSpecification testUser, WorkspaceApi workspace
     assertEquals(HttpStatusCodes.STATUS_CODE_FORBIDDEN, readerCannotDeleteBucket.getCode());
     logger.info("Failed to delete bucket {} directly as reader as expected", bucketName);
 
-    // TODO(PF-633): Owners and writers can actually delete buckets due to workspace-level roles
-    //  included as a temporary workaround. This needs to be removed as we transition onto WSM
-    //  controlled resources.
-
-    // // Owner also cannot delete the bucket directly
-    // StorageException ownerCannotDeleteBucket =
-    //     assertThrows(
-    //         StorageException.class,
-    //         () -> ownerStorageClient.get(bucketName).delete(),
-    //         "Workspace owner was able to delete a bucket directly!");
-    // assertEquals(HttpStatusCodes.STATUS_CODE_FORBIDDEN, ownerCannotDeleteBucket.getCode());
-    // logger.info("Failed to delete bucket {} directly as owner as expected", bucketName);
+    // Writer also cannot delete the bucket directly
+    StorageException writerCannotDeleteBucket =
+        assertThrows(
+            StorageException.class,
+            () -> readerStorageClient.get(bucketName).delete(),
```

Comment:
Should this be `writerStorageClient`?

Reply:
I'm not quite sure what I was thinking here - there is no writer in this test, just the workspace reader and owner. good catch!